.circleci/config.yml

```diff
@@ -155,7 +120,7 @@ jobs:
           root: /workspace
           paths: bin
 
-  check-hashes:
+  lint:
```

Comment:
Minor nit, but to me `lint` is not more descriptive of what this does than `check-hashes`

Reply:
`check-hashes` was no longer accurate (it also now checks for skew in various files) so I wanted to change it to something more generic, `lint` is a more general "checks" type word. Happy to take other suggestions though. I'm going to merge though since this is only a nit and I want to be around to keep an eye on the resulting master CI run.

Reply:
I couldn't come up with a better name...so yes, lint is fine too